src/config/user-setting.json

```diff
@@ -65,7 +65,7 @@
     {
         "name": "datalocation",
         "mainText": "Backup Location",
-        "value": "~/translationStudio",
+        "value": "~/BTT Writer",
```

Comment:
Is space ok here?

Reply:
Good catch.  We should probably make this a dash too.

Reply:
Fixed.